template/appveyor.yml

```diff
@@ -0,0 +1,43 @@
+environment:
+  matrix:
+  - julia_version: 0.7
+  - julia_version: 1
```

Comment:
Can you please remove the 0.7? To be consistent with the travis config. Then I will merge.

Reply:
for sure.